src/sentry/web/urls.py

```diff
@@ -583,6 +584,11 @@
         TeamAvatarPhotoView.as_view(),
         name="sentry-team-avatar-url",
     ),
+    url(
+        r"^sentry-app-avatar/(?P<avatar_id>[^\/]+)/$",
+        SentryAppAvatarPhotoView.as_view(),
+        name="sentry-app-avatar-url",
+    ),
```

Comment:
This + the file in `sentryapp_avatar.py` is such a weird pattern to serve images, kinda interesting. TIL 👍